generator/test/src/generator/TemplateGeneratorTests.scala

```diff
@@ -0,0 +1,62 @@
+package swam
+package generator
+
+import java.nio.file.Paths
+import java.util.concurrent.Executors
+
+import cats.effect.{Blocker, IO}
+import swam.runtime.Engine
+import utest.{TestSuite, Tests, test}
+
+import scala.concurrent.ExecutionContext
+
+/**
+  * @author Javier Cabrera-Arteaga on 2020-03-06
+  */
+object TemplateGeneratorTests extends TestSuite {
```

Comment:
Still not convinced this is a test. There is no check performed in the test cases.

Reply:
Removed test